src/keycloak-theme/app.config.ts

```diff
@@ -17,6 +17,6 @@ export const appConfig: ApplicationConfig = {
         provideRouter(THEME_ROUTES, withPreloading(PreloadAllModules)),
         provideI18nThemeDefinedMessages(),
         provideKcContext(),
-        provideHttpClient()
+        provideHttpClient(withInterceptorsFromDi())
```

Comment:
What is withInterceptorsFromDi?

Reply:
a feature that includes class-based interceptors configured using a multi-provider in the current injector into the configured HttpClient instance.
in angular we have two ways to configure interceptors, the other is: https://angular.dev/api/common/http/withInterceptors